sdks/python/apache_beam/ml/inference/tensorrt_inference.py

```diff
@@ -0,0 +1,281 @@
+#
+# Licensed to the Apache Software Foundation (ASF) under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to You under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+# pytype: skip-file
+
+import logging
+import sys
+from typing import Any
+from typing import Dict
+from typing import Iterable
+from typing import Optional
+from typing import Sequence
+from typing import Tuple
+
+import numpy as np
+
+import tensorrt as trt
+from apache_beam.io.filesystems import FileSystems
+from apache_beam.ml.inference.base import ModelHandler
+from apache_beam.ml.inference.base import PredictionResult
+from cuda import cuda
+
+TRT_LOGGER = trt.Logger(trt.Logger.INFO)
+
+logging.basicConfig(level=logging.INFO)
```

Comment:
Are these logging changes here just for debugging purposes while we are iterating on the PR?

See related: 

https://github.com/apache/beam/blob/01fa485b31bbaf649873eb27623539e2e2b27cf2/sdks/python/apache_beam/pipeline.py#L160
https://github.com/apache/beam/blob/01fa485b31bbaf649873eb27623539e2e2b27cf2/sdks/python/apache_beam/runners/worker/sdk_worker_main.py#L95

Reply:
TensorRT needs its own logger type. Regular logger was requested [here](https://github.com/apache/beam/pull/22131#discussion_r919051725).

Reply:
Got it. Understood re TensorRT logger. 
Regarding regular logger: I think we could instantiate it without overriding default logging settings (e.g. forcing logs to be Info), so that users can continue to customize desired default logging levels.

Typically the pattern is to define the logger like so: https://github.com/apache/beam/blob/ac7bb2bb95a6a47cbcf0bbe30d04eb46c2e8b9a8/sdks/python/apache_beam/io/textio.py#L47

Will that work?

Reply:
Sure, I'll put in this one liner to replace current logger stuff.